### PR TITLE
Fix typo in getCombinationIdsAction annotation and simplify js selector

### LINKED ISSUE #30843: BO - Button Bulk actions is hidden in PPV2 combinations
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

Button Bulk actions is hidden

### Expected behavior

_No response_

### Steps to reproduce

- FIRST CASE 
1. Go  to  Advanced Parameters >  New & Experimental Features > New product page - Single store 
2. Go to Catalog>Products>Create new product with combinations
3. Go to Combinations page >generate XX combinations >check the checkbox >select all results (XX) >SEE ERROR: the bulk actions button is hidden 

[Untitled_ Jan 12, 2023 3_48 PM (1).webm](https://user-images.githubusercontent.com/115152460/212118392-70cab8a6-686d-477b-bce1-440e299713b5.webm)

- SECOND CASE 
all the steps before > select result on this page (10) > then reselect all the results (XX) > SEE ERROR: it's not the same counts, still gives (10/XX)   combinations 
                                
bug

   

### PrestaShop version(s) where the bug happened

develop

### PHP version(s) where the bug happened

7.4

### If your bug is related to a module, specify its name and its version

_No response_

### LINKED ISSUE #30958: Product page v2 combination bulk actions button disabled when selecting all results
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

In Product page v2 combinations list the bulk actions button is disabled when selecting all results (note it seems to not work only when seleting all results. Other cases like "all in page" works well).
Screenshot 2023-01-19 at 17 44 47

### Expected behavior

when all results are selecte (same as with any result selected) the bulk action should be active.

### Steps to reproduce

1. go to product page v2 edit product with combinations
2. Go to combinations tab and see the list
3. In upper left corner there is a dropdown arrow which allows select option "select all results()", click it
4. See error - the "bulk actions" button above is still disabled.

You can try to select any other column one by one or "select all in page" option and you will see that the button is activated

### PrestaShop version(s) where the bug happened

develop

### PHP version(s) where the bug happened

_No response_

### If your bug is related to a module, specify its name and its version

_No response_

### PR DESCRIPTION
**Pull request by zuk3975**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | see https://github.com/PrestaShop/PrestaShop/issues/30843
| Type?             | bug fix
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | refer to #30843
| Fixed ticket?     | Fixes #30843
| Related PRs       | If theme, autoupgrade or other module change is needed to make this change work, provide a link to related PRs here.
| Sponsor company   | Your company or customer's name goes here (if applicable).

### CODE REVIEW

**Review by kpodemski — APPROVED** [review]

File: admin-dev/themes/new-theme/js/pages/product/components/combination-list/bulk-choices-selector.ts [hunk 1]
```diff
@@ -114,7 +114,7 @@ export default class BulkChoicesSelector {
         return;
       }
 
-      const isBulkSelectAll = checkbox.matches(`${CombinationMap.bulkSelectAll}, ${CombinationMap.bulkSelectAllInPage}`);
+      const isBulkSelectAll = checkbox.matches(CombinationMap.commonBulkAllSelector);
```
Comment (zuk3975) [inline admin-dev/themes/new-theme/js/pages/product/components/combination-list/bulk-choices-selector.ts:117]: this change actually not related to the fix, I just found while searching for the error cause and simplified it, it still works as inteded as both (`bulk select all` and `bulk select all in page`) buttons intentionally has that common class for that reason.

File: src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/CombinationController.php [hunk 2]
```diff
@@ -347,8 +347,7 @@ public function getListAction(int $productId, ProductCombinationFilters $combina
     /**
      * @AdminSecurity("is_granted('update', request.get('_legacy_controller'))")
      *
-     * @oaram int $productId
-     *
+     * @param int $productId
```
Comment (zuk3975) [inline src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/CombinationController.php:350]: 👿

**Review by jolelievre — APPROVED** [review]
Thanks @zuk3975

**Review by HanaRebaiQA — APPROVED** [review]
Hello @zuk3975 

I have checked your PR. The issue is well fixed.

Here is my feedback  : 

- Check the automated tests: OK
- Check Case 1 of the issue : OK :heavy_check_mark: 
- Check Case 2 of the issue: OK :heavy_check_mark: 
- Check with demo products : OK
- Check with new created products : OK
- Check with multistore: OK
- Check with different languages (RTL and no RTL) : OK
- Check with different browsers (Chrome & Firefox) : OK


image

image


So, It's QA :heavy_check_mark: 

Thank You!

**Pull request merged**